code/src/App.js

```diff
@@ -1,9 +1,94 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/media-has-caption */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
+import React, { useState } from 'react';
+import Header from './components/Header';
+import Name from './components/Name';
+import Love from './components/Love';
+import Favorite from './components/Favorite';
+import Result from './components/Result';
+import Footer from './components/Footer';
+import './app.css'
+import slowMotionAudio from './audio/slowmotion.mp3';
 
 export const App = () => {
+  const [question, setQuestion] = useState(0); // Start with question 0
+  const [name, setName] = useState('');
+  const [love, setLove] = useState('');
+  const [favorite, setFavorite] = useState('');
+
+  const handleQuestionIncrease = () => {
+    setQuestion(question + 1);
+  }
+
+  const handleRestartSurvey = () => {
+    setQuestion(0); // Go back to question 0
```

Comment:
Will definitely "borrow" this feature to get my restart to totally reset without keeping previous answers. :-)